### PR TITLE
Documentation: Fix code example in CustomNoOptionsMessage.js

### PR DESCRIPTION
**Pull request by cseas**
Change incorrectly passed custom component into styles to a styles property.

The custom `msgStyles` added to the `NoOptionsMessage` custom component example [on this page](https://react-select.com/components) are not showing up because the custom styles are added to `NoOptionsMessage` property instead of the correct one `noOptionsMessage`.

This fix will hopefully make the custom styles show up in the example.

P.s. Please note I was unable to run the docs website locally to verify this change. I've opened issue #4232 for this.

### CODE REVIEW

**Comment by cseas** [comment]
Update: The Netlify deploy preview shows that the fix works and makes the custom styles show up in the example:
Selection_087

**Comment by cseas** [comment]
Is anyone going to review this?

**Comment by Methuselah96** [comment]
Probably not here. But feel free to submit this on [my fork](https://github.com/react-select-oss/react-select).

**Comment by cseas** [comment]
@flexdinesh @JedWatson @bladey @gwyneplaine @mitchellhamilton

**Comment by ebonow** [comment]
@cseas unfortunately very few of us actually have permission to submit a PR. I will create another thread to better address how we can actually work to move the project forward just so it has a bit more transparency and that we're not bothering everyone subscribed to this.

**Comment by bladey** [comment]
Thanks for the PR @cseas - can you please add a changeset to this PR or give me write access so I can add one in so I can get it merged.

**Comment by cseas** [comment]
@bladey Thank you for the response! I think I'll have to make a new PR since I deleted the original fork. Anyway, how do I add a changeset?

The [CHANGELOG](https://github.com/JedWatson/react-select/blob/master/docs/CHANGELOG.md) seems to be automatically generated and [CONTRIBUTING](https://github.com/JedWatson/react-select/blob/master/CONTRIBUTING.md) has no info about how to document a change.

**Comment by bladey** [comment]
From what I can see (I'm new to this), if you have a look at the `.changeset` folder locally there README inside - https://github.com/JedWatson/react-select/tree/master/.changeset

Running `yarn changeset` allows you to select patch, minor or major version, with a summary provided. This then gets added to a new PR, like this one - https://github.com/JedWatson/react-select/pull/3980

Once this is merged a release on NPM is published and the changeset notes are provided here - https://github.com/JedWatson/react-select/releases

This should definitely be documented going forward. That changelog is only related to docs it seems.

I'm happy to open up a new PR, or if you're happy to open a new PR with the above changeset instructions, let me know!

**Comment by ebonow** [comment]
@bladey this only pertains to the documentation so there likely wouldn't need to be a version bump. The code example in the documentation was using TitleCasing instead of camelCasing for the component styles api, so maybe we simply remove this from the milestones and merge it in today?

**Comment by Methuselah96** [comment]
I agree with @ebonow, changes that don't change the package don't need a change set or a version bump, so this PR should be good to merge as-is. It would be nice if we could re-run the end-to-end test to get that passing (which I assume was broken because of https://github.com/JedWatson/react-select/pull/4294) or @cseas could just make a new PR, which might be easiest at this point.

**Review by ebonow — APPROVED** [review]
Good job @cseas updating documentation example

**Comment by ebonow** [comment]
I am unable to PR this since the CI was broken at the time. I have created a new PR https://github.com/JedWatson/react-select/pull/4316 to pull in these changes.

**Pull request closed without merging**
